main/networking/Ansible.js

```diff
@@ -6,86 +6,76 @@ import _ from 'lodash';
 
 import RendererBridge from '../RendererBridge';
 import { updateConsole } from '../../renderer/actions/ConsoleActions';
-import {
-  ansibleDisconnect,
-  infoPerMessage,
-  updateCodeStatus,
-} from '../../renderer/actions/InfoActions';
+import { ansibleDisconnect } from '../../renderer/actions/InfoActions';
 import { updatePeripherals } from '../../renderer/actions/PeripheralActions';
-import { robotState, Logger, defaults } from '../../renderer/utils/utils';
-import FCObject from './FieldControl';
-
-const DawnData = (new protobuf.Root()).loadSync(`${__dirname}/ansible.proto`, { keepCase: true }).lookupType('DawnData');
-const { StudentCodeStatus } = DawnData;
-
-const RuntimeData = (new protobuf.Root()).loadSync(`${__dirname}/runtime.proto`, { keepCase: true }).lookupType('RuntimeData');
-const Notification = (new protobuf.Root()).loadSync(`${__dirname}/notification.proto`, { keepCase: true }).lookupType('Notification');
+import { Logger, defaults } from '../../renderer/utils/utils';
+
+/**
+ * UDP Recv (Runtime -> Dawn)
+ * Device Data Array (sensors), device.proto
+ */
+const RecvDeviceProto = (new protobuf.Root()).loadSync('protos/device.proto', { keepCase: true }).lookupType('DevData');
+
+/**
+ * UDP Send (Dawn -> Runtime)
+ * Gamepad Data, gamepad.proto
+ */
+const SendGamepadProto = (new protobuf.Root()).loadSync('protos/gamepad.proto', { keepCase: true }).lookupType('GpState');
+
+/**
+ * TCP Recv (Runtime -> Dawn)
+ * Challenge Data (output values), text.proto
```

Comment:
I should mention this will I remember: the format of the challenge data is going to be basically one payload entry per challenge input / result. So if there are three functions in the coding challenge, and the inputs are 1, 2, and 3, the challenge data will have 3 strings in its `payload` field, the first element being the string `"1"`, the second element being the string `"2"`, and the third element being the string `"3"`.... hopefully that makes sense? Sorry, I should have clarified this earlier.